ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -398,6 +398,20 @@ CH_SS_SCHWTC_CHWSWISOVM_CHWDPM_CWDT_REFSM_REFPM_IGM_DXSWC:
   - IGM
   - DXSWC
 
+CH_SS_SCHWTC_CHWSWISOVM_CHWDPM_CHWDT_CHDX2SC_RCKTM2X:
```

Comment:
Probably don't need `CHWDT` or `SCHWTC`, as the points are covered in `CHDX2SC`. If this is water chilled why are there no condensing water points?

Reply:
It is an air cooled chiller after all, fixed.